specification.markdown

```diff
@@ -409,12 +409,16 @@ If the path contains an EBMLPathAtomRecursive part, the EBML Element can occur w
 
 As an example, a `path` of `1*(\Segment\Info)` means the element Info is found inside the Segment elements at least once and with no maximum iteration. An element SeekHead with path `0*2(\Segment\SeekHead)` may not be found at all in its Segment parent, once or twice but no more than that.
 
+The `@path` value MUST be unique within the EBML Schema. The `@id` corresponding to this `@path` MUST NOT be used in another EBML Element with the same EBMLParentPath as this `@path`.
```

Comment:
> I think 'used' is not clear enough as it could imply no elements with maxoccurs >1. I suggest:  `The @id value corresponding to this @path MUST NOT be defined for use within another EBML Element` ...

Reply:
I responded to my own comment with https://github.com/cellar-wg/ebml-specification/pull/310/commits/629ff4b8dbad177c131aa94f6b55c00a43c32696